.github/workflows/integtest.yaml

```diff
@@ -57,6 +57,24 @@ jobs:
       - name: tsc
         run: npx tsc
         working-directory: ${{ matrix.app_to_test }}
+      - name: Test redis cluster
+        if: matrix.app_to_test == 'ts-redis-cluster'
```

Comment:
rather than a test-specific `if:`, what about creating another optional hook (similar to`test/integ_test_hooks/pre-deploy.sh`)?

Reply:
would prefer not to because that requires a lot of steps, where as for this we are just using someone elses actions which sets everything up for us. We would have to download the redis-cli, go into each dir, start the server, then go create the cluster, and then. do the testing to make sure its ready

Reply:
Ah, that makes sense!